src/views/Home/components/PrizeTracks/PrizeTracks.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react"
+import "./PrizeTracks.scss"
+import HealthHacks from "./SVGS/HealthHacks"
+import BeginnerHacks from "./SVGS/BeginnerHacks"
+import EarthHacks from "./SVGS/EarthHacks"
+import GoldenStateHacks from "./SVGS/GoldenStateHacks"
+import JusticeHacks from "./SVGS/JusticeHacks"
+import UIUXHacks from "./SVGS/UIUXHacks"
```

Comment:
For SVGs there's a different way you can implement this which I think is much cleaner and simpler than creating a component for each of these. With an svg file you can write `import { ReactComponent as HealthHacks } from "path/to/svg"` and do a similar thing to what you did here. You wouldn't need to create all these tsx files and it's a lot quicker. I'll link an example of this method from our codebase if you need [reference](https://github.com/CruzHacks/cruzhacks-2022-website/blob/e16cd59de9b19ca089b5ef0113971c72a898c17b/src/components/Footer/Footer.tsx#L5-L7).